File: java/object-oriented programming/interface/interface 05/attorney.java
package com.mycompany.mavenproject1;

public class Advogado extends Funcionario {
    private String oab;

    public Advogado(String oab, String nome, String cpf, String rg, Endereco endereco, Setor setor, Sexo sexo, double salario, String dataNascimento) {
        super(nome, cpf, rg, endereco, setor, sexo, salario, dataNascimento);
        this.oab = oab;
    }
    
    @Override
    public double getSalarioFinal() {
       double getSalarioFinal = 0;
       getSalarioFinal += super.salario;
       return getSalarioFinal;
    }
    
    public String getOab() {
        return oab;
    }

    public void setOab(String oab) {
        this.oab = oab;
    }

    @Override
    public String toString() {
        return  super.toString() +
                "\nOab: " + oab;
    }
}

File: python/projects/project 04/models/attorney.py
from models.employee import Employee

class Advogado:
    def __init__(self, funcionario: Employee, oab: str) -> None:
        self.funcionario = funcionario
        self.oab = oab

    def __str__(self) -> str:
        return (
                f"\nAdvogado"
                f"\n{self.funcionario}"
                f"\nOAB: {self.oab}"
                )